operatorconfig/moduleconfig/authorization/v2.0.0-alpha/deployment.yaml

```diff
@@ -416,6 +416,116 @@ spec:
     kind: Issuer
     group: cert-manager.io
 ---
+# Controller
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: controller
+  namespace: <NAMESPACE>
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: controller
+rules:
+  - apiGroups: ["csm-authorization.storage.dell.com"]
+    resources: ["csmroles"]
+    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+  - apiGroups: ["csm-authorization.storage.dell.com"]
+    resources: ["csmroles/status"]
+    verbs: ["get", "update", "patch"]
+  - apiGroups: ["csm-authorization.storage.dell.com"]
+    resources: ["csmroles/finalizers"]
+    verbs: ["update"]
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["csm-authorization.storage.dell.com"]
+    resources: ["csmtenants"]
+    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+  - apiGroups: ["csm-authorization.storage.dell.com"]
+    resources: ["csmtenants/status"]
+    verbs: ["get", "update", "patch"]
+  - apiGroups: ["csm-authorization.storage.dell.com"]
+    resources: ["csmtenants/finalizers"]
+    verbs: ["update"]
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["create", "patch"]
+  - apiGroups: ["csm-authorization.storage.dell.com"]
+    resources: ["storages"]
+    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+  - apiGroups: ["csm-authorization.storage.dell.com"]
+    resources: ["storages/status"]
+    verbs: ["get", "update", "patch"]
+  - apiGroups: ["csm-authorization.storage.dell.com"]
+    resources: ["storages/finalizers"]
+    verbs: ["update"]
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: controller
+subjects:
+  - kind: ServiceAccount
+    name: controller
+    namespace: <NAMESPACE>
+roleRef:
+  kind: ClusterRole
+  name: controller
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller
+  namespace: <NAMESPACE>
+  labels:
+    app: controller
+spec:
+  replicas: 1
```

Comment:
Can we configure the number of replicas from the CR? This would be relevant for the controller and other deployments in Auth (tenant, storage, proxy, role, etc).

Reply:
Yes, we can have a parameter as replicas in the YAML file like we have for redisReplicas. Should I include a new parameter? @tdawe